7_react/react2/week1/vite-project/src/components/UserSearchResult.jsx

```diff
@@ -0,0 +1,23 @@
+import { useContext } from "react";
+import { userContext } from "./UserSection";
+
+function UserSearchResult() {
+  const { query, loading, error, searchResult } = useContext(userContext);
+
+  return (
```

Comment:
Consider simplifying the conditional logic. You could use a helper function or separate the outputs using if conditions, like so:

```jsx
if (loading) return <div className="loading">Loading...</div>;
```